esrally/telemetry.py

```diff
@@ -568,6 +568,134 @@ def record(self):
                 self.metrics_store.put_doc(doc, level=MetaInfoScope.cluster, meta_data=shard_metadata)
 
 
+class ShardStats(TelemetryDevice):
+    """
+    Gathers different shard per index stats every minute.
+    """
+
+    internal = False
+    command = "shard-stats"
+    human_name = "Shard Stats"
+    help = "Regularly samples _cat/shards"
```

Comment:
The API we use should be an implementation detail and we should probably not mention it here (at best in the user docs).